### PR TITLE
Add slow parsing detection to EOF layout fuzzing

### PR DESCRIPTION
**Pull request by shemnon**
## PR description

Add CLI flags and fuzzing logic to enable "slow" parsing to be a
loggable error.

### Thanks for sending a pull request! Have you done the following?

- [ ] Checked out our [contribution guidelines](https://github.com/hyperledger/besu/blob/main/CONTRIBUTING.md)?
- [ ] Considered documentation and added the `doc-change-required` label to this PR [if updates are required](https://wiki.hyperledger.org/display/BESU/Documentation).
- [ ] Considered the changelog and included an [update if required](https://wiki.hyperledger.org/display/BESU/Changelog).
- [ ] For database changes (e.g. KeyValueSegmentIdentifier) considered compatibility and performed forwards and backwards compatibility tests

### Locally, you can run these tests to catch failures early:

- [ ] unit tests: `./gradlew build`
- [ ] acceptance tests: `./gradlew acceptanceTest`
- [ ] integration tests: `./gradlew integrationTest`
- [ ] reference tests: `./gradlew ethereum:referenceTests:referenceTests`

### CODE REVIEW

File: testfuzz/src/main/java/org/hyperledger/besu/testfuzz/EofContainerSubCommand.java [hunk 1]
```diff
@@ -196,55 +216,54 @@ private void extractFile(final File f, final File initialCorpus) {
...
+                    } catch (IOException e) {
+                      throw new RuntimeException(e);
+                    }
+                  }
+                  if (value.toLowerCase(Locale.ROOT).startsWith("ok")) {
+                    passHappened.set(true);
+                  } else if (value.toLowerCase(Locale.ROOT).startsWith("err")) {
+                    failHappened.set(true);
+                  } else {
+                    // unexpected ouput: trigger a missmatch
```
Comment (macfarla) [inline testfuzz/src/main/java/org/hyperledger/besu/testfuzz/EofContainerSubCommand.java:250]:
```suggestion
                    // unexpected output: trigger a mismatch
```

File: testfuzz/src/main/java/org/hyperledger/besu/testfuzz/EofContainerSubCommand.java [hunk 2]
```diff
@@ -196,55 +216,54 @@ private void extractFile(final File f, final File initialCorpus) {
...
+    Map<String, String> resultMap =
+        fuzzingClients.stream()
+            .parallel()
+            .map(
+                client -> {
+                  Stopwatch stopwatch = Stopwatch.createStarted();
+                  String value = client.differentialFuzz(eofUnderTestHexString);
+                  stopwatch.stop();
+                  long elapsedMicros = stopwatch.elapsed(TimeUnit.MICROSECONDS);
+                  if (elapsedMicros > timeThresholdNs
```
Comment (macfarla) [inline testfuzz/src/main/java/org/hyperledger/besu/testfuzz/EofContainerSubCommand.java:232]: are we comparing microseconds with nanoseconds?

Comment (shemnon) [reply testfuzz/src/main/java/org/hyperledger/besu/testfuzz/EofContainerSubCommand.java:232]: Micros.  Only variable name was off.

File: testfuzz/src/main/java/org/hyperledger/besu/testfuzz/EofContainerSubCommand.java [hunk 3]
```diff
@@ -196,55 +216,54 @@ private void extractFile(final File f, final File initialCorpus) {
...
+                    } catch (IOException e) {
+                      throw new RuntimeException(e);
+                    }
+                  }
+                  if (value.toLowerCase(Locale.ROOT).startsWith("ok")) {
+                    passHappened.set(true);
+                  } else if (value.toLowerCase(Locale.ROOT).startsWith("err")) {
+                    failHappened.set(true);
+                  } else {
+                    // unexpected output: trigger a missmatch
```
Comment (macfarla) [inline testfuzz/src/main/java/org/hyperledger/besu/testfuzz/EofContainerSubCommand.java:250]:
```suggestion
                    // unexpected output: trigger a mismatch
```

**Review by macfarla — APPROVED** [review]

**Pull request merged**